doc/changes/changes_1.2.4.md

```diff
@@ -0,0 +1,38 @@
+# The Spark Exasol Connector 1.2.4, released 2023-??-??
```

Comment:
I assume updating dependencies only makes sense, when we release the spark connector -- please correct me if I'm wrong. 

Releasing would then require to define a release date. 
But in some cases I also created another branch eg. "release/1.2.4" and PR just for adjusting the release date in the changes file.
Maybe this is more convenient?

Reply:
I am not planning release yet,  I would like to investigate and fix also the #128, then release together with that.

Reply:
👍